README.md

```diff
@@ -26,11 +26,11 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 # Latest Best Practices and News
 
-- **✅ New best practice:** 7.1: [Don't block the event loop](#7-draft-performance-best-practices) by Keith Holliday
+- **:tada: Node.js best practices reached 40k stars**: Thank you to each and every contributor who helped turning this project into what it is today! We've got lots of plans for the time ahead, as we expand our ever-growing list of Node.js best practices even further.
 
-- **🇷🇺 Russian translation:** The amazing Alex Ivanov has just published a [Russian translation](/README.russian.md)
+- **:rocket: Continuously Improving**: We've been working hard on a new section about [using npm ci](https://github.com/goldbergyoni/nodebestpractices/pull/627) to preview the dependency state in production environments
```

Comment:
Maybe let's approve the 2 PRs and then your title could be:

2 new best practices

Reply:
Which two PRs?

Reply:
I think that would be #627 and #633. The first one is merged

Reply:
Added links to the new items, once the second PR is merged it will be usable 👍